amqp-0-9-1/doc/p_dev_js_client_amqp.md

```diff
@@ -1647,66 +1646,6 @@ var log = function(message) {
 
 You can also combine one or more of these programming styles.
 
-Migrate JavaScript Applications to KAAZING Gateway 5.0
```

Comment:
@pkhanal - please verify that this 3.x to 4.x migration content is not relevant to migrating 4.x to 5.0.x. Basically, I just deleted it.